src/universal/modules/welcome/containers/Welcome/Welcome.js

```diff
@@ -32,16 +49,27 @@ const mapStateToProps = state => ({
 export default class WelcomeContainer extends Component {
   static propTypes = {
     dispatch: PropTypes.func,
+    invitees: PropTypes.array,
+    inviteesRaw: PropTypes.string,
+    preferredName: PropTypes.string,
+    teamName: PropTypes.string,
     welcome: PropTypes.shape({
-      preferredName: PropTypes.string,
```

Comment:
oh fo sho! i was thinking about keeping it in its own duck in case someone exited halfway through & wanted to come back & pick up right where they left off, but that still depends on redux not throwing a warning about reducers that don't exist yet.